openapi-spec/swagger.yaml

```diff
@@ -3632,61 +4295,180 @@ components:
         key2: false
         key3:
           key31: value31
-    VolumeSpec:
-      description: 'Volume is an block device created by storage service, it can be
-        attached to physical machine or virtual machine instance. '
+    VolumeReqSpec:
```

Comment:
Are all the parameters included in Volume Request Spec required?

Reply:
NO. Please see carefully(In swagger.yaml there will be required keyword and in editor * mark will be there against each parameter.) For example, see the line no: 4302